recipes-omnect/iot-client-template-rs/iot-client-template-rs_0.5.0.bb

```diff
@@ -6,26 +6,26 @@ inherit cargo
 # DEFAULT_PREFERENCE = "-1"
 
 # how to get iot-client-template-rs could be as easy as but default to a git checkout:
-# SRC_URI += "crate://crates.io/iot-client-template-rs/0.4.21"
+# SRC_URI += "crate://crates.io/iot-client-template-rs/0.5.0"
 SRC_URI += "git://github.com/omnect/iot-client-template-rs.git;protocol=https;nobranch=1;branch=main"
-SRCREV = "d95819fd85397d2258c75dd94c2ab4b5e7732a07"
+SRCREV = "462b3125be487fba03ca7dc6cacc9cb164146cd8"
 S = "${WORKDIR}/git"
```

Comment:
new version is current HEAD of main but not that of tag 0.5.0 which references commit "Update CHANGELOG.md" with hash 3d2a711ad063821c0f92e7b0766ba95e41ed003f from 24.02.2022.
I guess that something went wrong during setting of tag 0.0.5?

Reply:
fixed. (obviously I didn't push the tag correctly)